component/authentication-endpoint/src/main/webapp/x509CertificateError.jsp

```diff
@@ -57,6 +57,21 @@
                     errorMessage = resourceBundle.getString("not.valid.certificate");
                 } else if (errorCode.equalsIgnoreCase("17003")) {
                     errorMessage = resourceBundle.getString("fail.validation.certificate");
+                } else if (errorCode.equalsIgnoreCase("17004")) {
+                    errorMessage = resourceBundle.getString(
+                            "x509certificateauthenticator.alternativenames.regex.multiplematches.error.message");
```

Comment:
it is better if we can encode the error code in message key. That helps to search the error code with grep like tools.
e.g. x509certificateauthenticator.alternativenames.regex.multiplematches.code.17004.error.message

Reply:
done